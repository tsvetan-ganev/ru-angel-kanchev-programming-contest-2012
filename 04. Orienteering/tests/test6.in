24 5
1705Karachorov m 08:42:50 09:35:58
2001Zhecheva w 08:39:13 09:25:05
1204Doganov m 09:41:41 10:33:15
1504Nedyalkova w 09:29:55 10:25:48
0205Ivanov m 08:34:37 09:24:36
0904Sirakov m 08:27:46 09:15:03
0701Nikolova w 10:19:19 11:12:42
1603Kamenarov m 08:13:12 08:56:27
2104Mihaylov m 10:13:15 11:17:00
1305Todorov m 10:29:08 11:32:33
1101Abadzhiev m 09:35:53 10:38:57
0804Belomazhev m 10:12:14 11:01:36
1005Dimitrov m 09:51:19 10:36:38
0501Dobrev m 09:38:52 10:42:49
1405Popunkyov m 10:53:13 11:50:51
0302Georgiev m 10:10:58 11:09:10
0601Zlateshka w 09:31:20 10:28:19
2201Atansova w 08:52:04 09:50:35
2302Keremedchieva w 09:31:26 10:33:40
2401Atanasov m 09:28:59 10:15:30
0103Dimitrova w 10:40:38 11:27:38
1802Shandurkova w 09:16:45 10:17:38
1901Zhecheva w 08:38:48 09:40:08
0402Stoyanov m 10:27:14 11:24:31