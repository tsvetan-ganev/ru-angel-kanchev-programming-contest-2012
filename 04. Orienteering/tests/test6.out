16 10 24 
20 01 07 
01 